src/dpos-user.ts

```diff
@@ -20,23 +20,27 @@ import {
 const log = debug('dpos-user')
 
 const coinMultiplier = new BN(10).pow(new BN(18))
-const ERC20GatewayABI = require('./mainnet-contracts/ERC20Gateway.json')
+const V2_GATEWAYS = ['oracle-dev', 'asia1']
```

Comment:
Err, no, we're not going to hardcode chain IDs in loom-js, can hardcode it in whatever uses loom-js.

Reply:
That was Matt's request after our discussion to simplify the process by which clients upgrade their logic.

Reply:
Of course it was.